app/retail/emails.py

```diff
@@ -322,6 +321,16 @@ def render_funder_payout_reminder(**kwargs):
     return response_html, response_txt
 
 
+def render_no_applicant_reminder(bounty):
+    params = {
+        'bounty': bounty,
+        'directory_link': '/users?skills=' + bounty.keywords.lower()
```

Comment:
@octavioamu / @PixelantDesigns so if the bounty has keywords `Python, javascript, MySQL , Moloch` 
The button would redirect to all those filters being applied => and no users would be shown as we would be searching for user who has all those 4 skills (especially Moloch)

We could match this against an exhaustive list at our end but even then we would end up with 3 skillm `Python, javascript, MySQL` and we might not have users with all 3 filters